app/javascript/components/map/assets/layers/loss.js

```diff
@@ -52,6 +52,13 @@ class Loss extends Canvas {
       .replace('{z}', z)
       .replace('{threshold}', this.options.threshold);
   }
+
+  setOptions(options) {
```

Comment:
Can this go on the parent layer?

Reply:
We have this method on the parent layer.
This is only an overwrite, for an specific behaviour for this layer.

Reply:
But it is repeated 3 times in loss, cover, and glads. When it could go in the parent canvas.js and still work fine?